content/blog/javascript-to-know-for-react.mdx

```diff
@@ -29,6 +29,7 @@ translations:
     author:
       name: Noelia Donato
       link: https://twitter.com/vamoacodear
+  - language: Español
```

Comment:
I was having trouble populating the blog post cache without this change. Maybe it was just me though 🙂  It seemed consistent with the structure here, so I kept it for reference in case it is needed.

Reply:
Thanks! That's definitely making the cache update fail.